force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -82,7 +94,9 @@ public with sharing class VariablesDataTypesOperators {
      * @return True if the number is even, False otherwise.
      */
     public static Boolean isEven(Integer num) {
-        return null; // Replace null with the variable you used to store the result
+        Integer a = Math.mod(num,2); // Mod() method of Math function returns the remainder of two parameters we passed into it.
+        Boolean result = system.assertEquals(0, a); // assertEquals method of the system static class will compare the expected result and the actual result and returns True / False
```

Comment:
This is not supposed to be used in regular classes. Asserts are for test classes.

Reply:
a = 0 ? return true : return false; 
 Am I correct using return statement in Ternary operator?